src/coreclr/src/jit/simd.cpp

```diff
@@ -3085,6 +3085,12 @@ GenTree* Compiler::impSIMDIntrinsic(OPCODE                opcode,
         }
         break;
 
+        case SIMDIntrinsicCeil:
+        case SIMDIntrinsicFloor:
+            op1    = impSIMDPopStack(simdType);
```

Comment:
This should have a check for the current SIMD level and should return `nullptr` if it isn't at least SSE4.1.

Essentially just `if (getSIMDSupportLevel() >= SIMD_SSE4_Supported)`, as is done in other places in this file.